### ISSUE TITLE
[BUG]: Scaling a weighted histogram does not appropriately scale the variance

### ISSUE THREAD

**Issue by matthewkenzie** [scikit-hep/boost-histogram#679]
**Describe the bug**

If I create a weighted histogram and then scale it I do not get the correct behaviour. It seems the variance (sum of squared weights) is simply scaled the same as the weights themselves. Therefore the relative error is not preserved. This is not the behaviour I would expect, and not the behaviour you get with Sumw2() on ROOT histograms, and I believe not the behaviour intended (I also don't think it's what is implemented in the C++ of `boost-histogram`)

**Steps to reproduce**

Simple to reproduce:

```python
import boost_histogram as bh
import numpy as np

h = bh.Histogram( bh.axis.Regular(50,0,1), storage=bh.storage.Weight() )
h.fill( np.random.uniform(size=10000) )

plt.errorbar( h.axes[0].centers, h.values(), h.variances()**0.5, fmt='ko' )

h /= h.sum().value

plt.errorbar( h.axes[0].centers, h.values(), h.variances()**0.5, fmt='ko' ) 

```
There is a notebook demonstrating this here: https://github.com/matthewkenzie/notebooks/blob/master/bh.ipynb

**Workaround**

The workaround is clunky and requires keeping hold of the total sumw (which I think is what `boost_histogram` should be doing anyway) and then dividing out again. I.e. **before** scaling:
```python
sumw = h.sum().value()
```
then scale:
```python
h /= h.sum().value
```
then plot with:
```python
plt.errorbar( h.axes[0].centers, h.values(), h.variances()**0.5 / sumw**0.5, fmt='ko' ) 
```

**Comment by HDembinski** [comment]
To add to this:

https://github.com/scikit-hep/boost-histogram/blob/develop/include/bh_python/accumulators/weighted_sum.hpp:65

Here we define multiplication for a weighted sum correctly, it should scale the variance with the scale of the value. But does not seem to be what is happening.

**Comment by henryiii** [comment]
However, `/=` is looked for and not found on that accumulator -> histogram, so this falls back to the Python view. But this both looks like it should be handled correctly, and also `.view() / x` should not be bugged, either. This is tested explicitly, and when I try your notebook commands locally, I get the correct behavior. What version of boost-histogram and numpy are you using?

**Comment by matthewkenzie** [comment]
I'm using boost-histogram v1.0.1 and numpy v1.21.4.

**Comment by matthewkenzie** [comment]
Probably should have tried this earlier but indeed an upgrade to boost-histogram 1.2.1 resolves the issue

**Comment by henryiii** [comment]
This was fixed in boost-histogram [1.0.2](https://github.com/scikit-hep/boost-histogram/releases/tag/v1.0.2) and backported to [0.13.2](https://github.com/scikit-hep/boost-histogram/releases/tag/v0.13.2), released in April. The fix was in #559. Any version newer than 1.0.1 will work for you.

**Comment by matthewkenzie** [comment]
Thanks. Apologies for the noise.
